designcompose/build.gradle.kts

```diff
@@ -74,11 +75,8 @@ dependencies {
     api(libs.androidx.lifecycle.runtime)
     api(libs.androidx.lifecycle.livedata.core)
     implementation(libs.accompanist.flowlayout)
-    implementation(libs.grpc.auth)
-    implementation(libs.grpc.stub)
-    implementation(libs.grpc.okhttp)
-    implementation(libs.grpc.protobufLite)
```

Comment:
Do we not need these grpc dependencies anymore, but still need grpc.stub?

Reply:
Yes. We haven't fully cleaned up the code that used GRPC yet. I was removing those dependencies, then realized that and added Stub back.